iocage/cli/set.py

```diff
@@ -33,12 +33,27 @@
     max_content_width=400, ), name="set", help="Sets the specified property.")
 @click.argument("props", nargs=-1)
 @click.argument("jail", nargs=1)
-@click.option("--plugin", "-P",
-              help="Set the specified key for a plugin jail, if accessing a"
-                   " nested key use . as a separator."
-                   "\n\b Example: iocage set -P foo.bar.baz=VALUE PLUGIN",
-              is_flag=True)
-def cli(props, jail, plugin):
+# @click.option("--plugin", "-P",
+#               help="Set the specified key for a plugin jail, if accessing a"
+#                    " nested key use . as a separator."
+#                    "\n\b Example: iocage set -P foo.bar.baz=VALUE PLUGIN",
+#               is_flag=True)
+# def cli(props, jail, plugin):
+def cli(props, jail):
     """Get a list of jails and print the property."""
+    jail = Jail.Jail(jail)
     for prop in props:
-        ioc.IOCage(jail, skip_jails=True).set(prop, plugin)
+
+        delete = False
+        try:
```

Comment:
The property might not contain `=` and thus splitting will not return two properties. If this happens we know the user wants to unset the property. Will refactor!